shared/utility/math/angle.hpp

```diff
@@ -165,7 +167,7 @@ namespace utility::math::angle {
 
         // Find alpha such that $\sin(\alpha) = a\_$ and $\cos(\alpha) = b\_$, which is possible because $a\_^2
         // + b\_^2 = 1$
-        float alpha = atan2(a_, b_);
+        float alpha = atan2f(a_, b_);
```

Comment:
this should have been changed to `std::atan2`, not to `atan2f`
The std versions are overloaded such that they will return the type that's put into them.